otdot/Druid-React22k-demo
# enabled Blog(Content)

In this PR:
 - enabled

To test:
 - run: `git clone`
 - run: `make fresh`
 - See that Structure>Views>Blogs is enabled
